### PR TITLE
Add position history to person page

### PR DESCRIPTION
**Pull request by maradragan**
This adds a previous positions section to the person details page.

Implements NCI-Agency/anet#568

### CODE REVIEW

File: src/main/java/mil/dds/anet/beans/Person.java [hunk 1]
```diff
@@ -37,6 +41,8 @@
 	private String domainUsername;
 		
 	private Optional<Position> position;
+
+	List<PersonPositionHistory> previousPositions;
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/beans/Person.java]: Make it private and supply a setter `setPreviousPositions`.

File: src/main/java/mil/dds/anet/database/PersonDao.java [hunk 2]
```diff
@@ -209,5 +226,25 @@ public Void inTransaction(Handle conn, TransactionStatus status) throws Exceptio
...
+				pph.setStartTime(pph.getCreatedAt());
+				if (pphPrev != null) {
+					pphPrev.setEndTime(pph.getStartTime());
+				}
+				pphPrev = pph;
+			}
+			// Remove all null entries
+			l = l.stream().filter(pph -> (pph != null && pph.getPerson() != null)).collect(Collectors.toList());
+			return l;
+		});
```
Comment (gjvoosten) [inline src/main/java/mil/dds/anet/database/PersonDao.java]: This inner block appears quite similar to the inner block in `PositionDao.getPositionHistory`, extract into a common method?

Comment (maradragan) [reply src/main/java/mil/dds/anet/database/PersonDao.java]: Yes, I saw that, but I had no idea where such a common method should be placed. Should I add it to the PersonPositionHistory bean maybe? Or make a specific util for it?

Comment (gjvoosten) [reply src/main/java/mil/dds/anet/database/PersonDao.java]: `PersonPositionHistory.java` seems like a good place.

**Pull request merged**